content/resources/csc108.md

```diff
@@ -0,0 +1,44 @@
+---
+title: CSC108
+icon: /icons/csc108.svg
+desc: Introduction to Computer Programming 
+link: csc108
+tags:
+  - utm-cs
+---
+
+# CSC108: Introduction to Computer Programming
+#### Prerequisite: None
+
+<grid-1-x-2 title="Fall 2020 Class Website" img-src="https://i.imgur.com/mthOVtT.png" link="https://mcs.utm.utoronto.ca/~108/index.shtml" desc="All credits to Michael Liut" button="Check it out!"></grid-1-x-2>
+
+### What is CSC108?
+CSC108 is an introductory computer programming course and is open to all students, whether or not they have any previous experiences in computer programming. 
+This course will teach the basics of programming, such as writing and analyzing algorithms, as well as tracing programs.
+
+After this course, students will gain the ability to create simple programs to their liking, such a calculator!
+### Topics in CSC108:
+- Python Arithmetics
+- Variables, Expressions and Statements
+- Elementary Data Types (Int, Bool, etc)
+- In Depth study on Strings
+- Conditional Statements
+- Functions and Classes
+- Data structures (Lists, Dictionaries, Tuples)
+- File Input and Output
+- Unit Testing
+- Sorting Algorithms
+- Time Complexity of Algorithms
+- Introduction to Object Orientated Programming
+
+### Resources:
+<grid-1-x-2 title="Beginner Tutorial to Python" :reversed="true" img-src="https://www.freecodecamp.org/news/content/images/2020/07/how-is-python-used-v2.png" link="https://developers.google.com/edu/python" desc="Get a head start on Python with Google for Education!" button="Check it out!"></grid-1-x-2>
+
+<grid-1-x-2 title="Practice makes perfect!" img-src="https://ictslab.com/wp-content/uploads/2019/03/d1326ca6cca8038cd115a061b4e2b3bc.png" link="https://developers.google.com/edu/python" desc="Multiple beginner exercises to improve your coding skills" button="Try it out!"></grid-1-x-2>
+
+### What courses are avaliable after CSC108:
+#### CSC148: Introduction to Computer Science
+A more indepth study on Python with harder concepts such as recursion.
+A Highly recommand course if students are looking to study more in computer science!
```

Comment:
Perhaps we can change this to "A highly recommended course for students looking to increase their knowledge in computer science"